Backpack/TabBarController/Classes/BPKTabBarController.h

```diff
@@ -26,6 +26,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 IB_DESIGNABLE @interface BPKTabBarController : UITabBarController
 
+/**
+*  Creates a new BPKTabBarController instance with a custom UITabBar
+*
+*  @param tabBar Custom UITabBar subclass to be used as the default tabBar
```

Comment:
Technically this is an instance of UITabBar, right?

```suggestion
*  @param tabBar Custom UITabBar instance to be used as the default tabBar
```